src/lavague/defaults.py

```diff
@@ -91,6 +91,48 @@ def default_get_selenium_driver() -> SeleniumDriver:
 
         driver = webdriver.Chrome(service=webdriver_service, options=chrome_options)
         return SeleniumDriver(driver)
+    
+    def evaluation_get_selenium_driver() -> SeleniumDriver:
+        """Extra options to make the driver more static for evaluation purposes."""
+        from selenium import webdriver
+        from selenium.webdriver.chrome.service import Service
+        from selenium.webdriver.common.by import By
+        from selenium.webdriver.chrome.options import Options
+        from selenium.webdriver.common.keys import Keys
+        import os.path
+
+        chrome_options = Options()
+        chrome_options.add_argument("--headless")  # Ensure GUI is off
+        chrome_options.add_argument("--no-sandbox")
+        chrome_options.add_argument("--window-size=1600,900")
+        chrome_options.add_experimental_option("prefs", {
+            "profile.managed_default_content_settings.javascript": 2
+        })
+        chrome_options.add_argument('--proxy-server=127.0.0.1:9999')
+
+        homedir = os.path.expanduser("~")
```

Comment:
I don't think we need a new function for this, we can keep it in the default get driver method like this and if evaluation is true, we will add those new options.
```python
def default_get_selenium_driver(evaluation=False):
    if evaluation:
        add options
    


```